app/controllers/org_admin/phases_controller.rb

```diff
@@ -29,15 +29,20 @@ def edit
       if !phase.template.latest?
         flash[:notice] = _('You are viewing a historical version of this template. You will not be able to make changes.')
       end
-      section = params.fetch(:section, nil)
-      render('container',
-        locals: { 
-          partial_path: 'edit',
-          template: phase.template,
-          phase: phase,
-          sections: phase.sections.order(:number).select(:id, :title, :modifiable),
-          current_section: section.present? ? Section.find_by(id: section, phase_id: phase.id) : nil
-        })
+      # User cannot edit a phase if its a customization so redirect to show
+      if phase.template.customization_of.present?
```

Comment:
I think this is too generic. A customizable template might have phases that do not belong to the customization_of template and therefore we should offer the possibility to edit them, am I right?